.github/script/get-prev-version

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+const exec = require('child_process').execSync;
+
+const versions = exec('npm view amazon-chime-sdk-js versions --json').toString().trim().split("\n");
```

Comment:
amazon chime js sdk? Or react library?

Reply:
Roster test doesn't need this script. Deleted the file.